docs/openapi.yaml

```diff
@@ -17,40 +17,18 @@ paths:
       parameters:
         - in: path
           name: index
-          description: "Index to search for resource in."
+          description: "Index to search for resources in."
```

Comment:
I think this needs re-writing as it is not clear to me what this means, not because it is wrong, but the terms are all generic. I'd guess "Index for NVA-resources"